jekyll/_cci2/api-permissions.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: classic-docs
+title: "API Permissions"
+short-title: "API Permissions"
+description: "How to Add and Change API Permissions on CircleCI"
+order: 20
+---
+
+To use the CircleCI API
+or view details about a build,
+you will need API tokens with the appropriate permissions.
+This document describes the types of API tokens,
+as well as how to create and delete them.
+
+## Overview
+
+There are two types of API token
+you can create within CircleCI.
+
+  1. **Personal**:
+  These tokens are used to interact with the CircleCI API
+  and grant full read and write permissions.
+  2. **Project**:
+  These tokens provide information about specific projects
+  and only grant read permissions.
+  Project tokens have three scope options: _Status_, _Build Artifacts_, and _All_.
+      - _Status_ tokens grant read access to the project's build statuses.
+      Useful for [embedding status badges]({{ site.baseurl }}/2.0/status-badges/).
+      - _Build Artifacts_ tokens grant read access to the project's build artifacts.
+      - _All_ tokens grant read access for the project's build statuses and build artifacts.
+
+**Note**:
+Once created,
+API tokens cannot be modified.
+The only way to change existing tokens
+is to delete and recreate them.
+
+### Creating a Personal API Token
+
+  1. In the CircleCI application,
+  go to your [User settings](https://circleci.com/account).
+  2. Click [Personal API Tokens](https://circleci.com/account/api).
+  3. Click the **Create New Token** button.
+  4. In the **Token name** field,
+  create a memorable name for the token.
+  5. Click the **Add API Token** button.
+  6. After the token appears,
+  copy and paste it to another location.
+  You will not be able to view the token again.
+
+To delete a personal API token,
+click the X in the **Remove** column.
+
+### Creating a Project API Token
+
+  1. In the CircleCI application,
+  go to your project's settings
+  by clicking the gear icon next to your project.
+  2. In the **Permissions** section,
+  click on **API Permissions**.
+  3. Click the **Create Token** button.
+  4. Choose a scope from the dropdown menu.
+  5. In the **Token Label** field,
+  create a memorable label for the token.
+  6. Click the **Add Token** button.
+
+To delete a project API token,
+click the X in the **Remove** column.
+When the confirmation window appears,
+click the **Remove** button.
+
+## What Next?
+
+Now that you know
+how to create API tokens,
+you could...
+
```

Comment:
^^ this tone feels different than our usual - is that intended? (curious also why you are writing in such short lines - poetry?)

Reply:
Ha. It is probably a bit too casual. The line spacing is something called [semantic linefeeds](http://rhodesmill.org/brandon/2012/one-sentence-per-line/). I've been using it to make editing and review easier.